waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -1339,10 +1352,23 @@
     (when (nil? (str/index-of waiter-config-token ","))
       waiter-config-token)))
 
+(defn add-pre-stop-config-for-bypass-service
+  "Returns a new vector of container configurations that include the desired preStop command and environment variables
+   that will be used by the preStop command to know how long to delay the SIGTERM signal to the container."
+  [container-configs pre-stop-cmd force-sigterm-secs sigterm-grace-period-secs]
+  (mapv #(-> %
+             (assoc-in [:lifecycle :preStop :exec :command] pre-stop-cmd)
+             (assoc :env (get % :env []))
+             (update :env conj {:name "WAITER_BYPASS_FORCE_SIGTERM_SECS" :value (str force-sigterm-secs)})
+             (update :env conj {:name "WAITER_BYPASS_SIGTERM_GRACE_PERIOD_SECS" :value (str sigterm-grace-period-secs)}))
```

Comment:
If a service configured these, would we respect those values? I think we should.